stellar_contract_sdk/src/testing/host/mod.rs

```diff
@@ -8,19 +8,9 @@
 
 pub mod fs;
 pub mod mem;
-use core::fmt::Debug;
 
 use crate::{Object, Val};
 
-impl Debug for Val {
```

Comment:
For some reason I couldn't leave this here, it kept complaining about a conflict. I saw the status one lives alongside the type, so I figured that was okay and did the same here.